pkgs/applications/virtualization/virt-manager/default.nix

```diff
@@ -20,20 +21,24 @@ python2Packages.buildPythonApplication rec {
   nativeBuildInputs = [ wrapGAppsHook intltool file ];
 
   buildInputs =
-    [ libvirt-glib vte virtinst dconf gtkvnc gnome3.defaultIconTheme avahi
+    [ libvirt-glib vte dconf gtkvnc gnome3.defaultIconTheme avahi
       gsettings_desktop_schemas libosinfo
-    ] ++ optional spiceSupport spice_gtk;
+    ]
+    ++ optional spiceSupport spice_gtk
+    ++ optional openstackSupport virtinst;
```

Comment:
keep the unconditional dep on ```virtinst```, it is harmless.

it depends on ```glanceclient``` too, so it is enough to pass ```glanceclient=null``` to both ```virtmanager``` and ```virtinst``` to get rid of broken openstack in both packages

also adding the ```openstackSupport``` option is a bit strange. it could be either ```false``` or broken, so there is no option to choose.